### PR TITLE
Provide support for WRK usage of COPY MAID

### PR DESCRIPTION
**Pull request by temanbrcom**
@VitGottwald have fun.
@Nurkambay, @niharikasw91 please, review.

### CODE REVIEW

**Review by grianbrcom — APPROVED** [review]
:shark:

**Review by niharikasw91 — APPROVED** [review]

**Comment by VitGottwald** [comment]
Thank you @temanbrcom ! I tried it out and it looks like the field suffix is appended rather than replacing the suffix from the copybook.

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopyMaidWithWRK.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Broadcom.
```
Comment (grianbrcom) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopyMaidWithWRK.java:2]: Here is an old year.

**Review by grianbrcom — CHANGES_REQUESTED** [review]
All tests are with the previous year.

**Review by zeibura — CHANGES_REQUESTED** [review]

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopyMaidWithWrkShowsErrorAfter77.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  @Test
+  void test() {
+    UseCaseEngine.runTest(
+        TEXT,
+        ImmutableList.of(new CobolText("BHTRGL-XBG", DialectType.MAID.name(), COPYBOOK_CONTENT)),
+        ImmutableMap.of(
+            "1",
+            new Diagnostic(
+                null,
+                "Cannot retrieve a suffix for WRK usage",
```
Comment (zeibura) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopyMaidWithWrkShowsErrorAfter77.java:51]: Cannot retrieve suffix for WRK usage

**Review by zeibura — CHANGES_REQUESTED** [review]

**Review by zeibura — APPROVED** [review]

**Comment by temanbrcom** [comment]
@VitGottwald awaiting your conclusion

**Comment by VitGottwald** [comment]
LGTM @temanbrcom, thank you! 🎉

**Pull request merged**